06_miscellaneous/README.md

```diff
@@ -2,13 +2,13 @@
 
 Learning goals:
 
-- TBD
+- Know the basics about several *other* important things concerning research software engineering: FAIRness of research data and research software, FLOSS licenses, versioning schemes, repository layout standards, and more.
 
 | Duration | Content |
 | --- | --- |
-| 15 minutes | [doi_zenodo_darus_slides.md](https://github.com/Simulation-Software-Engineering/Lecture-Material/blob/main/06_miscellaneous/doi_zenodo_darus_slides.md), [doi_zenodo_darus_demo.md](https://github.com/Simulation-Software-Engineering/Lecture-Material/blob/main/06_miscellaneous/doi_zenodo_darus_demo.md) |
-| 30 minutes | [floss_licenses_slides.md](https://github.com/Simulation-Software-Engineering/Lecture-Material/blob/main/06_miscellaneous/floss_licenses_slides.md) |
-| 10 minutes | [versioning_slides.md](https://github.com/Simulation-Software-Engineering/Lecture-Material/blob/main/06_miscellaneous/versioning_slides.md) |
-| 10 minutes | [repository_layout_slides.md](https://github.com/Simulation-Software-Engineering/Lecture-Material/blob/main/06_miscellaneous/repository_layout_slides.md) |
-| 15 minutes | [tools_alex_slides.md](https://github.com/Simulation-Software-Engineering/Lecture-Material/blob/main/06_miscellaneous/tools_alex_slides.md), [tools_alex_demo.md](https://github.com/Simulation-Software-Engineering/Lecture-Material/blob/main/06_miscellaneous/tools_alex_demo.md) |
+| 20 minutes | [doi_zenodo_darus_slides.md](https://github.com/Simulation-Software-Engineering/Lecture-Material/blob/main/06_miscellaneous/doi_zenodo_darus_slides.md) |
+| 20 minutes | [floss_licenses_slides.md](https://github.com/Simulation-Software-Engineering/Lecture-Material/blob/main/06_miscellaneous/floss_licenses_slides.md) |
+| 20 minutes | [versioning_slides.md](https://github.com/Simulation-Software-Engineering/Lecture-Material/blob/main/06_miscellaneous/versioning_slides.md) |
+| 20 minutes | [repository_layouts_slides.md](https://github.com/Simulation-Software-Engineering/Lecture-Material/blob/main/06_miscellaneous/repository_layout_slides.md) |
+| 20 minutes | [tools_alex_slides.md](https://github.com/Simulation-Software-Engineering/Lecture-Material/blob/main/06_miscellaneous/tools_alex_slides.md), [tools_alex_demo.md](https://github.com/Simulation-Software-Engineering/Lecture-Material/blob/main/06_miscellaneous/tools_alex_demo.md) |
```

Comment:
Do we still want to keep Alex's name in the file names? Authorship is pretty clear from the Git history. We could give a more meaningful name here.

Reply:
I thought about just removing these slides eventually. But that's sth for a future PR.